student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly ensure the input is in the right order in the super initializer, what's happening right now is that age is being saved in place of name because the parameter order is different compared to the one defined in the person class.